### DISCUSSION TITLE
Add Support for MMOCore / MMOItems and NBT Tagged Items

### DISCUSSION THREAD

**Discussion by kiloRs** [Zrips/CMI#3882] [Feature Requests]
My request is to add support for MMOCore and MMOItems.
MMOItems: (NBT TAG SUPPORT)
- Add conditions for ranks
- Custom Crafting
- Add to Kits
MMOCore:
- Conditions for ranks
- Conditions for being able to craft X item if your X class / level

Mythic Mobs:
- Conditions for ranks

**This could also be done by adding placeholderAPI support for ranks**

**Comment by roy0510** [comment]
Codition for Rank of MMOCore LEVEL PROFESSIONS

**Comment by CloudeLecaw** [comment]
I dont think the custom crafting will be added since cmi only Support vanilla unmodes items at the Moment as materials. Its an official Statement u also will find on the wiki
